clients/guzzle6-adapter.rst

```diff
@@ -32,6 +35,15 @@ Then create the adapter::
 
     $adapter = new GuzzleAdapter($guzzle);
 
+.. note::
+
+    You can also use the quicker `createWithConfig()` function::
```

Comment:
actually, why don't we explain createWithConfig as the default way? and then have a ..note:: that if you have a guzzle client already somehow configured can use the `new GuzzleAdapter` syntax.